lib/extensions/active_record/connection_adapters/table_definition.rb

```diff
@@ -0,0 +1,14 @@
+module Extensions::ActiveRecord::ConnectionAdapters::TableDefinition
+  def self.included(module_)
+    module_.alias_method_chain :userstamps, :compatibility
+  end
+
+  def time_bounded(*args)
+    datetime :valid_from, *args
+    datetime :valid_to, *args
+  end
+
+  def userstamps_with_compatibility(*args)
```

Comment:
I don't quite got you, why we need this helper?  since we're not using compatibility mode.